apps/win32/package.json

```diff
@@ -60,7 +60,7 @@
     "@wdio/sync": "5.20.1"
   },
   "peerDependencies": {
-    "@office-iss/react-native-win32": "0.62.3",
+    "@office-iss/react-native-win32": "0.62.6",
```

Comment:
You may be able to get rid of this, it's not really a peer dep. Not sure it's a dev dep either, though it seems like some things may have broken when removing it from dev deps and placing it as a dep in other places? Any idea what was going on there?

Reply:
No idea. I just tried it locally and it seems to be fine. I'll move it to dependencies.

Reply:
Actually just started seeing problems now. When removing it, the win32 package will no longer bundle. Adding it back to devDependencies fixes it.